src/daemons/rhsmcertd.c

```diff
@@ -152,6 +156,10 @@ log_update (int delay)
 	return TRUE;
 }
 
+int gen_random(int min, int max) {
+    return (rand() % max) + min;
```

Comment:
I think we need to check `RAND_MAX` here since if `max` doesn't divide evenly into `RAND_MAX` the numbers will have a bias.  Not that it's a huge deal, but if we are doing this to avoid clumping, then we probably want as even a distribution as possible.  See [here](http://www.azillionmonkeys.com/qed/random.html) and [here](http://stackoverflow.com/a/1202735/6124862)